src/feature/flow_chart_panel/views/ControlBar.tsx

```diff
@@ -215,6 +217,7 @@ const LoadButton = () => {
 
 const ControlBar = () => {
   const { states } = useSocket();
+  const { nodes, edges } = useFlowChartGraph();
```

Comment:
This hook is still being called in `ControlBar`.. I would recommend extracting the cancel button into a component and calling it in there instead, along with putting the keyboard shortcut hook inside of there.